docs/blog/zh/README.md

```diff
@@ -1,5 +1,5 @@
 # 博客
 
 
-- [KubeVela 正式开源：一个高可扩展的云原生应用平台与核心引擎](/blog/zh/kubevela-the-extensible-app-platform-based-on-open-application-model-and-kubernetes.md)
-- [如何在 20 分钟内给你的 K8s PaaS 上线一个新功能](/blog/zh/extend-kubevela-by-cuelang-in-20-mins.md)
\ No newline at end of file
+- [KubeVela 正式开源：一个高可扩展的云原生应用平台与核心引擎](/docs/blog/zh/kubevela-the-extensible-app-platform-based-on-open-application-model-and-kubernetes.md)
+- [如何在 20 分钟内给你的 K8s PaaS 上线一个新功能](/docs/blog/zh/extend-kubevela-by-cuelang-in-20-mins.md)
```

Comment:
If changed, will it work on the website? https://kubevela.io/#/blog/zh/

Reply:
The website is generate by the docs? Where the code, I fix it also.

Reply:
Use the same doc, so we can read it normally from website but error link in raw github link.

Reply:
I think it needs to be unified by repairing the official website configuration. For example, set website add prefix `/docs` to support. Or move blog page to the new repo to be independent.

Reply:
yes, we could discuss more in Dingtalk with @sunny0826